.github/workflows/icu_ci.yml

```diff
@@ -573,4 +573,57 @@ jobs:
           # To run all tests serially
           echo "Run all samples"
           pwd
-          PATH=$PATH:/tmp/icu_samples/bin LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/tmp/icu_samples/lib make check-samples-recursive                  
+          PATH=$PATH:/tmp/icu_samples/bin LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/tmp/icu_samples/lib make check-samples-recursive
+
+  # https://unicode-org.github.io/icu/processes/release/tasks/integration.html#verify-that-icu4c-tests-pass-without-collation-rule-strings
+  icu4c-without-collation-rule-strings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install hjson dependency
+        run: |
+          sudo apt install python3-pip
```

Comment:
use 'apt-get' to install instead of 'apt'. 
see https://askubuntu.com/questions/829864/is-apt-get-deprecated

Reply:
or you can avoid adding hjson by just use json. 
see   lstm-icu4j-build-and-test which use .github/lstm_for_th_my.json
or adaboost-icu4j-build-and-test which use .github/adaboost.json
you can still generate a icu4c/coll-norules.json on the fly like you did below, but you need to tweak it to json format from hjson (which is just slightly different)

Reply:
Done. Used `apt-get` instead of `apt`. The documentation for the BRS task used an example with hjson, and this CI job is just a direct translation of that, so I kept hjson unchanged.